LoopKitUI/CGMManagerUI.swift

```diff
@@ -22,6 +22,9 @@ public protocol CGMManagerUI: CGMManager, DeviceManagerUI {
     
     /// the completed percent of the progress bar to display in the status bar
     var cgmLifecycleProgress: DeviceLifecycleProgress? { get }
+    
+    /// gets the type of a glucose sample using the CGM manager managed glucose thresholds
+    func glucoseValueType(for glucose: GlucoseSampleValue) -> GlucoseValueType?
```

Comment:
"ValueType" is quite vague. Looking at what GlucoseValueType contains, maybe `GlucoseRangeCategory`?